.github/workflows/shellcheck.yml

```
@@ -0,0 +1,18 @@
+name: ShellCheck
```

```suggestion
name: Shell
```

Let's name it also `Shell` so in case we need to add more later.